src/de_main.c

```diff
@@ -3,7 +3,7 @@
 //           and it should be isolated from platform specific
 //           stuff when it's reasonable.
 //
-static void Game_AdvanceSimulation(AppState *app)
+static void Game_AdvanceSimulation(AppState *app, float time_step)
```

Comment:
Most probably this will get inlined and optimized but we could just make the timestep a #define constant to make sure the compiler can take advantage of it being constant.